src/components/Books.jsx

```diff
@@ -1,35 +1,37 @@
-import { useState } from "react";
-import { Link } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { removeBook } from "../redux/books/books";
 import AddBook from "./AddBook";
 import Book from "./Book";
 
-const data = [
-  {
-    id: 1,
-    author: "Kyle Simpson",
-    title: "You don't know JS"
-  },
-  {
-    id: 2,
-    author: "Chinua Achebe",
-    title: "Things fall apart"
-  }
-]
-
-
 function Books() {
-  const [books] = useState(data);
+  const booksList = useSelector((state) => state.booksReducer);
+  const dispatch = useDispatch();
+
+  const bookEl = booksList.map(({ id, title, author }) => (
```

Comment:
Putting this code here is an anti-pattern and unnecessary... Displaying a list of items can/should be done directly in the component for a number of reasons like performance and code quality...

If your aim is to make this into its own code then I suggest refactoring this to be a reusable component